src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -64,7 +64,7 @@
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
     // '&' not allowed in names
```

Comment:
Could you change the comment to reflect the changes in the tests 👍 . Maybe 
```java
//first character whitespace not allowed in names
```

Reply:
The first character white space issue isn't important because the parser will filter out white spaces at the front.

Reply:
Yes I understand that, would u like to rename the comment to match what you are testing?